run.sh

```diff
@@ -2,9 +2,6 @@
 source set_env.sh
```

Comment:
Do we need this and `set_env.sh`? Right now this script installs CoreNLP, which we don't want to do automatically anymore, and launches jupyter.